yaks/README.md

```diff
@@ -552,6 +552,28 @@ yaks = Yaks.new do
 end
 ```
 
+Yaks also has support for respecting the `include` query parameter (e.g.
+`include=posts.comments,account`), which is a behaviour you can include in
+your mappers:
+
+```ruby
+require "yaks/behaviour/optional_includes"
+
+class UserMapper < Yaks::Mapper
+  include Yaks::Behaviour::OptionalIncludes
+
+  has_many :posts
+  has_one :account
+end
+
+# ...
+
```

Comment:
`bundle exec rake rubocop` from the top level should work. Does it show up when you do `bundle exec rake -T`?